Makefile

```diff
@@ -0,0 +1,6 @@
+# IF your sockeye source directory lives in the same parent directory as sockeye-react
+# then you can use this makefile to build sockeye-react and update the static contents of sockeye
+update-sockeye:
```

Comment:
I dont know if you will like this addition or not.. for me it saved a lot of time.